deploy/charts/github-token-manager/values.yaml

```diff
@@ -0,0 +1,74 @@
+---
+## namespace: defaults to the namespace where the chart is deployed if not specified
+namespace: ~
+
+## crds
+##   install: true | false
+##     true: install the CRDs
+##     false: do not install the CRDs
+crds:
+  install: true
+
+## rbac
+##   create: true | false
+##     true: create the RBAC resources
+##     false: do not create the RBAC resources
+rbac:
+  install: true
+
+## service
+##   type: ClusterIP | NodePort | LoadBalancer | ExternalName
+##   ports: list of ports
+##     name: name of the port
+##     port: port number
+##     protocol: protocol
+##     targetPort: target port
+service:
+  type: ClusterIP
+  ports:
+    - name: https
+      port: 8443
+      protocol: TCP
+      targetPort: https
+
+## deployment
+##   replicas: number of replicas
+##   tolerations: list of tolerations (optional)
+##   nodeSelector: node selector (optional)
+##   containers: list of containers
+##     githubTokenManager:
+##       image:
+##         repository: image repository
+##         tag: image tag
+##     kubeRbacProxy:
+##       image:
+##         repository: image repository
+##         tag: image tag
+deployment:
+  hostNetwork: false
```

Comment:
If we're going to make this an option, then we need to make *all* the port bindings configurable too…

Reply:
Good point. I will remove this instead as adding overrides for all bindings adds unnecessary complexity for now...